server/routerlicious/packages/services-core/src/containerManager.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDocument } from "./document";
+
+export interface IContainerManager {
```

Comment:
Maybe I am not fully understand the entire design. Will FRS provide its own implementation of IContainerManager? If so, I don't see this interface is injected somewhere else, like the StorageNameProvider. Or it will be in the next PR?

Reply:
What is the purpose of this interface/class and why does it have to be introduced here in OSS ?